webpack.config.js

```diff
@@ -5,9 +5,10 @@ var webpack = require('webpack');
 var bourbon = require('bourbon').includePaths;
 var neat = require('bourbon-neat').includePaths;
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
+require('babel-polyfill');
 
 var config = {
-  entry: { app: ['./src/client.js'] },
+  entry: ['babel-polyfill', './src/client.js'],
```

Comment:
If we do this, do we need to server babel-polyfill off the produciton server? If so, where path does it need to be at? and lastly, do we need to move it from devDependencies to dependencies in package.json?